packages/client/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { WebpackManifestPlugin } = require("webpack-manifest-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: process.env.NODE_ENV,
+  entry: path.join(__dirname, "./src/index.tsx"),
+  output: {
+    path: path.resolve(__dirname, "../server/public/dist"),
+    publicPath: "/dist/",
+    filename: "bundle.js",
+  },
+  devServer: {
+    port: 8080,
+    static: {
+      directory: path.join(__dirname, "public/"),
+    },
+    proxy: [
+      {
+        context: ["/api"],
+        target: "http://localhost:8000",
+      },
+    ],
+    historyApiFallback: true,
+  },
+  module: {
+    rules: [
```

Comment:
A lot of the following `rules` section boils down to webpack incantations. There's probably optimizations/fixes to be done here, but it works for now.